### ISSUE TITLE
Clean out garage. Make room for one car?

### ISSUE THREAD

**Issue by emtrimble** [scott-joe/dreamland#12]

**Comment by scott-joe** [comment]
Yes, that's definitely part of the garage's design. Make room for 1 car and when someone needs to use the wood shop, the car can be moved out then returned when work has been completed. That's where the garage door opener comes into play.
